dali/operators/image/crop/random_crop_attr.h

```diff
@@ -28,6 +28,24 @@
 
 namespace dali {
 
+class RandomCropGeneratorWrap {
```

Comment:
What is this class for? It seems that it adds no functionality except maybe the call operator (which could be easily replaced with a lambda).